uwblueprint/ccbc
# Omar/update review service

## Notion ticket link

[Review Service: Update Review](https://www.notion.so/uwblueprintexecs/Review-Service-Update-Review-5c6374c816774d9688ea7b4d7aa6e4c0)



## Implementation description
* Added endpoint to update review by ID
* Added test
* Added Swagger docs




## Steps to test
1. Verify that all the tests are passing
2. Add some reviews to DB and try updating them




## What should reviewers focus on?
* 


## Checklist
